README.md

```diff
@@ -116,6 +116,18 @@ A more detailed description of the ways to filter resources:
    
    The tag filter can be negated by surrounding the regex with `NOT(...)`
 
+   Resources not matching tag can be removed with Untagged: true
```

Comment:
In my understanding, `Untagged: true` means resources should be deleted that are not tagged at all.

Thinking about it, I actually see three cases and we should cover them all:

1) Delete resources that are not tagged (`set of tags` is nil so-to-say)
2) Delete resources having some tags that match a list of given tag expressions (`foo: bar`, `boo: baz`), i.e., a resource must have both tag `foo: bar` and `boo: baz`.
3) Delete resources having some tags that do NOT match some given tag expressions (`foo: bar`, `boo: baz`), i.e., a resource's tag set must not include `foo: bar` nor `boo: baz`.

Any thoughts on this? :smiley: